server/aws-lsp-codewhisperer/package.json

```diff
@@ -33,30 +33,31 @@
     "dependencies": {
         "@amzn/codewhisperer-streaming": "*",
         "@aws-sdk/util-retry": "^3.374.0",
-        "@aws/language-server-runtimes": "^0.2.11",
         "@aws/chat-client-ui-types": "^0.0.6",
+        "@aws/language-server-runtimes": "^0.2.11",
         "@aws/lsp-fqn": "^0.0.1",
         "@gerhobbelt/gitignore-parser": "^0.2.0-9",
         "@smithy/node-http-handler": "^2.5.0",
         "adm-zip": "^0.5.10",
         "archiver": "^6.0.1",
         "aws-sdk": "^2.1403.0",
         "deepmerge": "^4.3.1",
+        "fastest-levenshtein": "^1.0.16",
         "got": "^11.8.5",
         "hpagent": "^1.2.0",
         "js-md5": "^0.8.3",
-        "fastest-levenshtein": "^1.0.16",
         "proxy-http-agent": "^1.0.1",
         "uuid": "^9.0.1",
         "vscode-uri": "^3.0.8"
     },
     "devDependencies": {
-        "@types/archiver": "^6.0.2",
-        "mock-fs": "^5.2.0",
         "@types/adm-zip": "^0.5.5",
+        "@types/archiver": "^6.0.2",
         "@types/uuid": "^9.0.8",
         "assert": "^2.1.0",
         "copyfiles": "^2.4.1",
+        "mock-fs": "^5.2.0",
```

Comment:
is mock-fs still used in tests?

Reply:
yes, we are still using it to test artifact generation part